aten/src/THCUNN/generic/SpatialFullDilatedConvolution.cu

```diff
@@ -86,16 +94,17 @@ void THNN_(SpatialFullDilatedConvolution_updateOutput)(
   THCUNN_assertSameGPU(state, 6, input, output, weight,
                        bias, columns, ones);
   THNN_(SpatialFullDilatedConvolution_shapeCheck)
-       (state, input, NULL, weight, bias, kH, kW, dH, dW, padH, padW, dilationH, dilationW, adjH, adjW);
+       (state, input, NULL, weight, bias, kH, kW, dH, dW, padH, padW, dilationH, dilationW, adjH, adjW, 0);
 
+  THArgCheck(!bias || THCTensor_(isContiguous)(state, bias), 5,
+             "bias tensor has to be contiguous");
   input = THCTensor_(newContiguous)(state, input);
   weight = THCTensor_(newContiguous)(state, weight);
-  bias = bias ? THCTensor_(newContiguous)(state, bias) : bias;
```

Comment:
Oh that's interesting. So updateOutput used to accept non-contiguous bias, but you've decided to tighten it up here because the forwards doesn't accept noncontiguous bias either, is that right?

Reply:
The `THNN_(SpatialFullDilatedConvolution_shapeCheck)` call used to check that `weight` and `bias` are contiguous. So the `newContiguous` call here doesn't really matter. :)